extensions-core/druid-bloom-filter/src/test/java/org/apache/druid/query/aggregation/bloom/BloomFilterAggregatorTest.java

```diff
@@ -91,6 +93,12 @@ public class BloomFilterAggregatorTest extends InitializedNullHandlingTest
   private static final Float[] FLOAT_VALUES1 = new Float[]{0.4f, 0.8f, 23.2f};
   private static final Long[] LONG_VALUES1 = new Long[]{10241L, 12312355L, 0L, 81L};
 
+  private static final Object[] ARRAY_VALUES = new Object[]{
+      new Object[]{1L, 2L},
+      new Object[]{3L, 4L},
+      new Object[]{0L, 1000L}
+  };
```

Comment:
Not sure how important it is, but I notice that you are not validating `null` or `new Object[]{1L, null}` and making sure that it handles null as expected.